www/doc/2.1/tutorial.html

```diff
@@ -44,8 +44,8 @@ <h2>Giving Herbie expressions</h2>
 
   <p>
     After a brief wait, your web browser should open and show you
-    Herbie's main window. The most important part of the page is this
-    bit:
+    Herbie's main window. Check installation if this doesn&apos;t
+    occur. The most important part of the page is this bit:
```

Comment:
Maybe we put "The most important part" in its own paragraph to help it stand out.